demos/node-classification/rgcn/rgcn-aifb-node-classification-example.ipynb

```diff
@@ -186,16 +176,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 16,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/Users/wil9dw/projects/stellargraph/stellargraph-4/stellargraph/core/experimental.py:88: ExperimentalWarning: RGCN is experimental: it has severe known bugs (see: https://github.com/stellargraph/stellargraph/issues/649, https://github.com/stellargraph/stellargraph/issues/677). It may be difficult to use and may have major changes at any time.\n",
+      "/Users/kieranricardo/PycharmProjects/origin-stellargaph/stellargraph/stellargraph/core/experimental.py:88: ExperimentalWarning: RGCN is experimental: it has severe known bugs (see: https://github.com/stellargraph/stellargraph/issues/649, https://github.com/stellargraph/stellargraph/issues/677). It may be difficult to use and may have major changes at any time.\n",
```

Comment:
Not related to this PR, but... did you know you've spelled stellargraph wrong in this directory name? 😝 

```suggestion
      "/Users/kieranricardo/PycharmProjects/origin-stellargraph/stellargraph/stellargraph/core/experimental.py:88: ExperimentalWarning: RGCN is experimental: it has severe known bugs (see: https://github.com/stellargraph/stellargraph/issues/649, https://github.com/stellargraph/stellargraph/issues/677). It may be difficult to use and may have major changes at any time.\n",
```

Reply:
Nothing gets past your reviews! I'm hopeless without spell check 😝